docs/documentation/Upgrading.md

```diff
@@ -131,7 +146,7 @@ new DataTable("table", {
         if (!tr.attributes) {
             tr.attributes = {}
         }
-        tr.attributes["data-name"] = rowValue[1].data
+        tr.attributes["data-name"] = rowValue.cells[1].data
```

Comment:
@SandroHc Given that this is an example on how to upgrade to version 6.0.x, it seems like this should probably not be changed, right? But maybe this changed example could go under the 8->9 upgrade notes?

Reply:
I've reverted it. Didn't mean to change it, but was in the middle of a fixing spree and didn't notice the file I was on.